### PR TITLE
🛠️ fix incorrect COCO class id loading

### PR DESCRIPTION
**Pull request by SkalskiP**
# Description

Draft of PR fixing issues mentioned https://github.com/roboflow/supervision/pull/150. It is an alternative to the https://github.com/roboflow/supervision/pull/169 solution.

### CODE REVIEW

**Comment by SkalskiP** [comment]
Hi, @hardikdava 👋🏻 did you have a chance to test this?

**Comment by hardikdava** [comment]
@SkalskiP Let me test and confirm your fix.

**Comment by hardikdava** [comment]
@SkalskiP Test summary:

Situation-1: `from_coco()` -> `to_coco()`
Works fine if `image_id` and `labels_id` starts with `0`

Situation-2: `from_yolo()` -> `to_coco()`
`image_id` and `labels_id` starts with `0` after saving as coco format. But no mixed up in categories.

**Comment by SkalskiP** [comment]
@hardikdava, as for case 2, do you think we would like `image_id` and `labels_id` to start from `1`?

**Comment by hardikdava** [comment]
@SkalskiP Yes, based on my research and description from coco, opencv, etc it is their standard format to start `image-id` and `label-id` starting from `1`. Once this is fixed then we are good to go.

File: supervision/dataset/formats/coco.py
Comment (hardikdava) [inline supervision/dataset/formats/coco.py:1]: @SkalskiP Can you make following changes then we are done with coco format. 

file: coco.py line:202 and 203
``
    image_id = 1
    annotation_id = 1
``

**Review by hardikdava — CHANGES_REQUESTED** [review]
Please make this small changes and we can close this.

**Comment by SkalskiP** [comment]
@hardikdava, that should be easy. I'll update `.to_coco` logic before merging.

**Comment by SkalskiP** [comment]
@hardikdava given the fact that @yclicc tested the code and it works as expected. (read more here https://github.com/roboflow/supervision/pull/150) I'll probably close https://github.com/roboflow/supervision/pull/169. Is it okey with you?

**Comment by hardikdava** [comment]
@SkalskiP Yes, please mark it as completed.

File: supervision/dataset/formats/coco.py
Comment (SkalskiP) [reply supervision/dataset/formats/coco.py:1]: done

**Review by hardikdava — APPROVED** [review]

**Pull request merged**